### PR TITLE
Adapted WebARonTango to have support for the same API as WebARonCore

### PR DESCRIPTION
**Pull request by judax**
Adapted both the code and the documentation to make WebARonTango compatible with WebARonARCore and WebARonARKit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,157 @@
...
+
+## <a name="DifferencesBetweenWebARonTangoAndOthers">Differences between WebARonTango and WebARonARCore/WebARonARKit</a>
+
+All of the prototypes are build on top of the WebVR 1.1 spec and for the basic AR capabilities (motion tracking, camera feed rendering and hit testing) they all share a common workflow. However, WebARonTango exposes some additional functionalities and lacks others compared to the ARCore and ARKit based projects.
+
+### What does WebARonTango have that WebARonARCore/WebARonARKit does not?
+
+* Depth point cloud support: WebARonTango provides access to the depth point cloud that the infrared camera in the Tango spec includes.
+
+* **ADF support**: For now, WebARonTango exposes a very simple access to the Area Description capabilities of Tango. Maybe in the future, WebARonARCore may expose this functionality too.
```
Comment (lincolnfrog) [inline README.md]: WebARonARCore AND WebARonARKit?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,157 @@
...
+
+* **Marker support**: For now, WebARonTango exposes QRCode and AR marker support. Maybe in the future, WebARonARCore may expose this functionality too.
```
Comment (lincolnfrog) [inline README.md]: Same as above

File: chromium/src/third_party/WebKit/Source/modules/vr/VRPassThroughCamera.cpp [hunk 3]
```diff
@@ -0,0 +1,83 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "modules/vr/VRPassThroughCamera.h"
+
+namespace blink {
+
+VRPassThroughCamera::VRPassThroughCamera(): m_width(0)
+	, m_height(0)
```
Comment (lincolnfrog) [inline chromium/src/third_party/WebKit/Source/modules/vr/VRPassThroughCamera.cpp]: This file is all tabs, not spaces :(

File: chromium/src/third_party/tango/libtango_chromium/TangoHandler.h [hunk 4]
```diff
@@ -45,6 +45,12 @@
 
 namespace tango_chromium {
 
+class Hit
+{
+public:
+	float modelMatrix[16];
```
Comment (lincolnfrog) [inline chromium/src/third_party/tango/libtango_chromium/TangoHandler.h:51]: This file is tabs, not spaces :(

**Review by lincolnfrog — APPROVED** [review]
Some style notes, otherwise looks good! I mostly looked through everything, looks like its merging a bunch of changes we made previously in addition to this excellent readme :D

File: README.md (hunk 1 above)
Comment (judax) [reply README.md]: For now it seems impossible as ARKit does not have ADF support :(. But I get your point, maybe not even mention it?

File: README.md (hunk 2 above)
Comment (judax) [reply README.md]: This is more likely to land sooner than ADFs. I will rewrite it to add WebARonARKit too.

**Comment by judax** [comment]
New patch with the fixes to the review.

**Pull request merged**